tests/Inspector/Http/snapshots/HttpCombinedRegistrationTest-tools_call-discovered_status_check.json

```diff
@@ -5,5 +5,8 @@
       "text": "System status: OK (discovered)"
     }
   ],
-  "isError": false
+  "isError": false,
+  "structuredContent": {
+    "result": "System status: OK (discovered)"
+  }
```

Comment:
I wonder a bit about this change here - it's not adding value to me. spec is unclear about it, but i think we shouldn't just add it always.

Reply:
@chr-hertel this has now been actioned, so structrued content will only be added if outputschema is present and explicility specified